acsf_tools.drush.inc

```diff
@@ -172,10 +172,24 @@ function drush_acsf_tools_ml() {
     unset($options['php']);
     unset($options['php-options']);
 
+    $profile = $options['profile'] ?? '';
```

Comment:
@nikunjkotecha I would suggest the profile argument to be a string of profiles comma separated. So we can do --profile=profile1,profile2

Reply:
+1 but I love the spirit of this pr

Reply:
updated